index.css

```diff
@@ -74,14 +158,42 @@ img {
   justify-content: end;
 }
 
+@media screen and (max-width: 1023px) {
+  .hero {
+    padding: 100px 20px 40px 20px;
+  }
+}
+
 .hero-text {
   text-align: center;
+  margin-bottom: 15px;
+}
+
+.hero-button-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.hero-button {
+  cursor: pointer;
+  font-size: 30px;
+  text-transform: uppercase;
+}
+
+@media screen and (max-width: 1023px) {
+  .hero-button {
+  font-size: 20px;
+  }
+}
+
+.hero-button:hover,
+.hero-button:focus {
+  color: var(--accent-color);
 }
 
 h1 {
-  /* font-family: Roboto, sans-serif; */
   text-align: center;
-
   font-size: 40px;
   font-weight: bolder;
```

Comment:
Please use numbers instead of 'bolder'
here and in 6 other places

Reply:
here is ok 
`.description {
  text-align: center;
  line-height: 1.5;
  letter-spacing: 0.03em;

  font-size: 25px;
  **font-weight: 300;**

  margin-left: auto;
  margin-right: auto;
  padding: 0 0 30px;
  margin: 0;
}`

Reply:
why?

Reply:
done

Reply:
you can add these font-weight numbers to variables (:root), if you wish))